src/main/java/com/laserfiche/repository/api/RepositoryApiClientImpl.java

```diff
@@ -127,9 +130,14 @@ public static RepositoryApiClient createFromUsernamePassword(String repositoryId
     @Override
     public void setDefaultRequestHeaders(Map<String, String> defaultRequestHeaders) {
         defaultHeaders = defaultRequestHeaders;
-        httpClient
+        if (!httpClient
                 .config()
-                .clearDefaultHeaders();
+                .getDefaultHeaders()
+                .containsKey("Accept-Encoding")) {
```

Comment:
i'm not sure if `setDefaultRequestHeaders()` needs to be changed. If the user chooses to override the default request headers with new defaults, then we should let them clear the existing default headers

If the user wants to keep the existing defaults, then they can call `getDefaultRequestHeaders()` and add their new default headers to the returned Map and set it with `setDefaultRequestHeaders()`

Reply:
ok reverted back